src/main/java/ca/intelliware/ihtsdo/mlds/config/CacheConfiguration.java

```diff
@@ -33,56 +31,51 @@ public class CacheConfiguration {
     @PersistenceContext
     private EntityManager entityManager;
 
-    @Inject
+    @Autowired
     private Environment env;
 
-    @Inject
+    @Autowired
     private MetricRegistry metricRegistry;
 
-    private static net.sf.ehcache.CacheManager cacheManager;
-    private static EhCacheCacheManager ehCacheManager;
+    private static CaffeineCacheManager cacheManager;
 
     @PreDestroy
     public void destroy() {
         log.info("Remove Cache Manager metrics");
-        SortedSet<String> names = metricRegistry.getNames();
-        for (String name : names) {
-            metricRegistry.remove(name);
-        }
-        //log.info("Closing Cache Manager");
-        //cacheManager.shutdown();
```

Comment:
@dmcgihtsdo While updating the cache manager the new code internally removing the records from metric registry so that's why we removed these lines. now we are added these lines again for safer side.